src/classes/STG_PanelCustomizableRollup_CTRL.cls

```diff
@@ -89,33 +106,40 @@ public with sharing class STG_PanelCustomizableRollup_CTRL extends STG_Panel {
     public virtual override PageReference saveSettings() {
 
         STG_SettingsManager_CTRL.idPanelCurrent = idPanel();
-        Customizable_Rollup_Settings__c crlpSettings = UTIL_CustomSettingsFacade.getCustomizableRollupSettings();
-        Savepoint sp = Database.setSavepoint();
         isEditMode = false;
-        try {
-            if (STG_Panel.stgService.stgCRLP.Customizable_Rollups_Enabled__c) {
-                String oldStatus = STG_Panel.stgService.stgCRLP.CMT_API_Status__c;
-                // Only start polling if it's their first time or the last deployment wasn't a success
-                if(String.isEmpty(oldStatus) || !oldStatus.contains('Succeeded')) {
+
+        Boolean settingHasChanged = (cachedSetting != STG_Panel.stgService.stgCRLP.Customizable_Rollups_Enabled__c);
+
+        if (settingHasChanged) {
+            // we only proceed with deployment/save if they changed the checkbox.
+
+            // if they hit edit and then they immediately hit save,
+            // we do NOT want to reset their CRLP customizations (if both were true)
+            // or run extraneous code (if both were false) so in that case we do nothing.
+            // they have essentially hit cancel.
+
+            Savepoint sp = Database.setSavepoint();
+            try {
+                if (STG_Panel.stgService.stgCRLP.Customizable_Rollups_Enabled__c) {
                     isPolling = true;
+                    jobId = CRLP_DefaultConfigBuilder_SVC.convertLegacyRollupsIntoCustomizableRollups();
+                    if (Test.isRunningTest()) {
+                        jobId = '123';
+                    }
+                } else {
+                    // The deployment method takes care of setting the Enabled flag to true if it succeeds.
+                    // In here, we only set it to false in the case on disabling CRLPs.
+                    stgService.saveAll();
                 }
-                // This method includes committing Customizable_Rollups_Enabled__c as true if the deployment succeeds
-                jobId = CRLP_DefaultConfigBuilder_SVC.convertLegacyRollupsIntoCustomizableRollups();
-                if (Test.isRunningTest()) {
-                    jobId = '123';
-                }
-            } else {
-                // The deployment method takes care of setting the Enabled flag to true if it succeeds.
-                // In here, we only set it to false.
-                stgService.saveAll();
+                UTIL_MasterSchedulableHelper.setScheduledJobs();
```

Comment:
If the Custom Setting isn't being set to true the deployment completes, then the call to `UTIL_MasterSchedulableHelper.setScheduledJobs();` probably needs to move because that has method has code to check for the current setting to determine which jobs to schedule (legacy or crlp rollups). Also, what if the user enables CRLP and the job fails. Are the new jobs scheduled anyway?